ckanext/charts/templates/scheming/form_snippets/chart_more_info_button.html

```diff
@@ -0,0 +1,77 @@
+<div class="form-group control-full control-large">
```

Comment:
adding some string translations in this file (`{{ _('xxx') }}`) might be a good idea?